coxeter/shapes/base_classes.py

```diff
@@ -107,6 +107,48 @@ def plot(self):
     def __str__(self):
         return repr(self)
 
+    def _plato_primitive(self, backend, **kwargs):
+        raise NotImplementedError("This shape does not have a plato primitive.")
```

Comment:
I'm anticipating that most people will be unfamiliar with plato, so perhaps a more informative error would just be to indicate that plotting is not supported (which can always be forwarded to devs who do know that this means that the `_plato_primitive` method must be implemented.
```suggestion
        raise NotImplementedError("This shape does not support plotting.")
```

Reply:
1. This is only raised when users call a method named `to_plato_scene`, so I think the existing error message here is more descriptive/helpful. Users should not be surprised to see the name "plato."
2. I think "does not support plotting" falsely indicates a deficiency in coxeter rather than the external library (plato) that coxeter is calling.

Reply:
Yes, that's a good point. It does make me wonder whether we should alias `to_plato_scene` with something like `plot` to make it easier for unfamiliar users who might not know what to look for if they don't find something that is actually called `plot`. I'm OK doing that in a separate PR if you'd like to just be done with this one for now, but I do think that would be useful for anyone (especially outside our group) who tries to use it.

Reply:
I think we can leave it as `to_plato_scene` for now. I think this is a case where explicit is better than implicit. Plotting is very generic, but this functionality is fairly specific to plato. I could see `plot` being an alias for creating a matplotlib figure, for instance.